client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -492,6 +492,21 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         MOBILE_APP_LOCALE("mobileAppLocale", "en_US", false),
 
         /**
+         * This parameter represents the appium device.
+         */
+        MOBILE_DEVICE("mobileDevice", "", false),
+
+        /**
+         * This parameter represents the appium device platform version.
+         */
+        MOBILE_PLATFORM_VERSION("mobilePlatformVersion", "", false),
```

Comment:
If we want to run the SAME test case on different platforms and different devices, we can not define these parameters in `MobileTest` annotation. 

if we remove them, Best case scenario, there will be methods with just one line, that call another method (the test case content), and two different classes for android and ios ( this part can be solved by setting groups in test case, but then we need to have to test suites).

another benefit of this is that now, you can define which device and platform you want your tests to run on from command line (useful when automating tests with continuous integration solutions)